WDL2CWL_example/GATK4/AddOrReplaceReadGroups.cwl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env cwl-runner
```

Comment:
This should go in the https://github.com/common-workflow-library/bio-cwl-tools/tree/release/GATK folder

Reply:
restructured the contents.